RNTester/js/LanguagesExample.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ * @providesModule LanguagesExample
+ */
+'use strict';
+
+
+var React = require('react');
+var ReactNative = require('react-native');
+var {
+  LanguagesAndroid,
+  Platform,
+  Text,
+} = ReactNative;
+
+exports.framework = 'React';
+exports.title = 'Languages';
+exports.description = 'Device languages';
+
+const examples = [
+  {
+    title: 'navigator.language',
+    render: function(): React.Element<any> {
+      return <Text>{navigator.language}</Text>;
+    },
+  },
+  {
```

Comment:
This has been fixed here: https://github.com/facebook/react-native/pull/14568/commits/b6e8b91fb5efd18a4789751a8687985fe2d375ad#diff-0468751a1a9708b31a37cde71e096450R20